### PR TITLE
📝 Add Contact page

### PR DESCRIPTION
**Pull request by bmiddha**
- Add contact page
- Add `/contact` to footer nav
- Add `SlackSignUpLink` component
- Add slack workspace links to config.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -79,15 +79,15 @@ module.exports = {
...
+                <a class="footer__link-item" href="${appConfig.discordServerLink}" target="_blank" rel="noreferrer noopener" aria-label="ACM@UIC Discord Server">
                   <img alt="Discord" src="https://img.shields.io/discord/652006495675875359?label=Discord&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2">
                 </a>
               `,
             },
             {
               // label: 'Slack',
               html: `
-                <a class="footer__link-item" href="https://uicacm.slack.com/signup" target="_blank" rel="noreferrer noopener" aria-label="ACM@UIC Slack Workspace">
+                <a class="footer__link-item" href="${appConfig.slackWorkspaceSignUpLink}" target="_blank" rel="noreferrer noopener" aria-label="ACM@UIC Slack Workspace">
```
Comment (clee231) [inline docusaurus.config.js:90]: For some reason, these links are still showing up as `/undefined` in the footer.
Perhaps the appConfig object isn't available here?
20210919_15h17m33s_grim
(Mouse was hovering over the "Slack" link in the footer. The Discord link also exhibits the same behavior.

**Review by clee231 — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (bmiddha) [reply docusaurus.config.js:90]: #71 fixes that

Comment (bmiddha) [reply docusaurus.config.js:92]: rebased the branch. should have the fix now that #71 merged.

Comment (clee231) [reply docusaurus.config.js:92]: Gah, derp. Why didn't I put 1 and 1 together. :facepalm: 
In that case, Laevoversion Gobony Tylopoda Malchite ([LGTM](https://wilt00.github.io/lgtm/))!

**Review by clee231 — APPROVED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (bmiddha) [reply docusaurus.config.js:92]: @clee231 not 2 and 2. Needed to put #71 and #72 together. 🤣

**Pull request merged**